newapi/ooni_api_uploader.py

```diff
@@ -204,10 +208,12 @@ def main():
             jsonl_s3path = (
                 f"raw/{tstamp[:8]}/{tstamp[8:10]}/{cc}/{testname}/{jsonlf.name}"
             )
-            upload_to_s3(s3, bucket_name, postcanf, postcan_s3path)
-            upload_to_s3(s3, bucket_name, jsonlf, jsonl_s3path)
-
-            update_db_table(db_conn, lookup_list, jsonl_s3path)
+            if conf.get("run_mode", "") == "DESTROY_DATA":
```

Comment:
This sounds like a very dangerous mode of operation to have. Should we maybe add some extra guards (like that you must run the tool with some special option) to prevent this happening by mistake?
At the very least, I think we should display a very big warning when the script is invoked, that it's in this mode of operation.

Reply:
Yes I see that, but since it's dependent on something which is inside of a file (that you don't see very clearly when you run it from the CLI), there is a the risk for a mistake.

Can we at least add a warning message to stderr/stdout when it's started in this mode of operation?

Reply:
The tool runs as a daemon and it's not meant to be ran manually from CLI.

Reply:
What about requiring a command line flag from the daemon init script, so it's clear from reading that script that it's going to behave this way.
My concern is just that we might accidentally have this value set in the config file and not notice it (it's in a config file that is distant from the how the tool is started).

Also the log message is printed only after the files have already been deleted, so we should move printing this further up in the call chain, perhaps at the beginning of the main.